src/main/java/io/github/dsse/models/IntotoEnvelope.java

```diff
@@ -0,0 +1,81 @@
+package io.github.dsse.models;
+
+import java.util.List;
+import java.util.Objects;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+
+/**
+ * Implementation of the a DSSE Envelope. The Envelope is the outermost layer of the attestation,
+ * handling authentication and serialization. The format and protocol are defined in DSSE and
+ * adopted by in-toto in ITE-5. It is a JSON object with the following fields: payloadType string,
+ * required
+ *
+ * <p>Identifier for the encoding of the payload. Always application/vnd.in-toto+json, which
```

Comment:
nit: perhaps it's because I'm not seeing the rendered JavaDoc, but this format confuses me a bit.  The <p> parts seem to reference individual fields, but I don't actually see them named.